app/controllers/avo/actions_controller.rb

```diff
@@ -73,12 +73,15 @@ def respond
               turbo_stream.close_action_modal,
               turbo_stream.flash_alerts
             ]
+          when :navigate_to_action
+            frame_id = Avo::ACTIONS_TURBO_FRAME_ID
+            src, _ = @response[:action].link_arguments(resource: @action.resource, **@response[:navigate_to_action_args])
+
+            render turbo_stream: turbo_stream.turbo_frame_set_src(frame_id, src)
           when :redirect
-            # Turbo redirect to the path
             render turbo_stream: turbo_stream.redirect_to(
               Avo::ExecutionContext.new(target: @response[:path]).handle,
-              nil,
-              @response[:redirect_args][:turbo_frame],
+              turbo_frame: @response[:redirect_args][:turbo_frame],
               **@response[:redirect_args].except(:turbo_frame)
             )
```

Comment:
Let's also update the DOCS about how to use `navigate_to_action` on the multi action flux 
https://docs.avohq.io/3.0/actions.html#:~:text=you%20may%20want%20to%20redirect%20to%20another%20action.%20

Will this still work for people who implemented it with the redirect method or should be referred as a breaking change?

Reply:
No. it will not work anymore.
That's a breaking change.
Do we know who originally asked for this?

Reply:
We had some discussion arround this:

Adding and maintaining a new action response ( `navigate_to_action` in this case ) VS Using the `redirect_to` method ( no maintenance need, only docs on how to use it on multi action flux )

This is the original request:
https://discord.com/channels/740892036978442260/1162459368827146292

I think more projects are using it, I remember sending the docs links on some discord threads.

Reply:
@adrianthedev just tested it locally with the city update action, it works with the `redirect_to` so it's not a breaking change, it's only another way of doing it

Reply:
It is if you run the action multiple times. It does something weird like emptying out the page.

Reply:
And that don't happen on main? There is a [similar issue with downloads](https://github.com/avo-hq/avo/issues/2385)